topics/statistics/tutorials/aberrant_pi3k_pathway_analysis/tutorial.md

```diff
@@ -0,0 +1,643 @@
+![GSE69822_samples_classification](../../images/aberrant_pi3k_pathway_analysis/external.png ") PI3K_OG classifiers applied to MCF10a cell lines dataset (GEO: GSE69822). The samples were either WT/WT+A66 (blue circles) or having a PIK3CA-H1074R/PTENKO mutation (orange circles).")
+
+> ### {% icon hands_on %} Hands-on: external sample evaluation with ERBB2_PIK3CA_KRAS_AKT1 model
+>
+> 1. {% tool [PAPAA: PanCancer external sample status prediction](toolshed.g2.bx.psu.edu/repos/vijay/pancancer_external_sample_status_prediction/pancancer_external_sample_status_prediction/0.1.9) %} with the following parameters:
+>    - {% icon param-file %} *"Classifier data"*: `classifier_summary.txt` (output of **PAPAA: PanCancer classifier** {% icon tool %})
+>    - {% icon param-file %} *"pancancer classifier coefficients"*: `classifier_coefficients.tsv` (output of **PAPAA: PanCancer classifier** {% icon tool %})
+>    - {% icon param-file %} *"external sample gene expression data"*: `GSE69822_pi3k_trans.csv` (Input dataset)
+>    - {% icon param-file %} *"given mutational status"*: `GSE69822_pi3k_sign.txt` (Input dataset)
+{: .hands_on}
+
+> ### {% icon details %} PanCancer Aberrant Pathway Activity Analysis `external_sample_pred_targene_classifier.py` inputs
+>    - \--classifier_summary: String of the location of classifier scores file
+>    - \--expression_file: File path for external sample expression data file (fpkm/rpkm/rlog/vlog/nlog values)
+>    - \--status_sign: Provide sign for tumor/mutant [1] or normal/WT [-1] along with sorted sample names
+{: .details}
+
+> ### {% icon comment %} Outputs
+>    - Model based classification scores distribution Plots for external normal and tumor samples ("targene_external_sample_predictions.pdf and targene_external_sample_predictions_1.pdf") 
+>    - Log file for script run and additional information
+{: .comment}
+
+## **PanCancer targene pharmacology**
+In this step we use the classifier derived cell line predictions and use them to evaluate pharmacological response of these cell lines. We plot log IC50 with classifier scores for each cell line and draw a correlation for drug response in absence or presence of targene mutations.
+
+![Afatinib_pharmacological_response](../../images/aberrant_pi3k_pathway_analysis/PI3K_OG_gdsc1_ccle_drug.png "Natural log fitted IC50 (Ln-IC50) values of Afatinib pharmacological response compared against PI3K_OG classifier scores of various cell lines. Cell lines with mutant (orange) or wild-type (blue) for ERBB2, KRAS, PIK3CA, AKT1 are indicated. The trend lines and p values are shown separately for wild-type or mutant cell lines. Cell lines treated with Afatinib, a known EGFR inhibitor".)
```

Comment:
I feel genes (such as ERBB2) should be linked for further reading.

Reply:
@anuprulez nice suggestion will add that